crates/optimism/evm/src/lib.rs

```diff
@@ -30,9 +30,23 @@ pub use error::OptimismBlockExecutionError;
 use revm_primitives::{Bytes, Env, OptimismFields, TxKind};
 
 /// Optimism-related EVM configuration.
-#[derive(Debug, Default, Clone, Copy)]
+#[derive(Debug, Default, Clone)]
 #[non_exhaustive]
-pub struct OptimismEvmConfig;
+pub struct OptimismEvmConfig {
+    chain_spec: ChainSpec,
```

Comment:
perhaps we even should use OpChainSpec here already

Reply:
Done